### ISSUE TITLE
Major version bump with no changelog entry?

### ISSUE THREAD

**Issue by supairish** [GetJobber/recurring_select#149]
Hello, just saw the 3.0 version release but don't see any documentation of what the changes where since 2.x

**Comment by jcw-** [comment]
Good point @supairish - looks like we're a bit behind on the changelog file, but we've been putting the details in the release artifact, the 3.0 release notes are here: https://github.com/GetJobber/recurring_select/releases/tag/v3.0.0
